cdk/script/ci

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+npm install -g yarn
```

Comment:
I'm fairly certain `yarn` is already available via `actions-setup-node`, so this would be a no-op / can be removed.